testsuite/tests/linear/should_fail/LinearErrOrigin.stderr

```diff
@@ -1,4 +1,8 @@
 
+LinearErrOrigin.hs:7:7: error:
+    • Nontrivial multiplicity equalities are currently not supported
+    • In an equation for ‘foo’: foo f x = f x
```

Comment:
Do we get this error every time we have a multiplicity error? That's unfortunate. Is there a way to silence these? Like, realising that `tc_subtype_ds` has failed?

Reply:
I don't see any way to do this; `tc_subtype_ds` is not failing, it's generating a constraint that can't be solved later.

Reply:
This is quite unfortunate. I guess it will have to do for now. But we will need a better solution soon.